### PR TITLE
Update vignette readmes for consistency

### PR DESCRIPTION
**Pull request by mackgalante**
Making format and layout changes to standardize all existing vignettes while updating links to github re: issue #96 
- fixed link references for `is about` throughout codebase
- updated formatting of mechanisms to standard
- updated formatting of moderators to standard
- updated message generation links, formatting, and organization

### CODE REVIEW

**Comment by mackgalante** [comment]
Left improving, goal gain, and social loss vignettes as is, updated vignettes aside from those three.

Ready for review!

File: vignettes/social_gain_vignette/readme.md [hunk 1]
```diff
@@ -75,32 +75,31 @@ This pathway describes the influence of feedback interventions informing the rec
...
+2. [Positive performance gap content <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000104)
+3. [Positive performance trend content <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000099)
+4. [Achievement content <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000112)
 
 Message preconditions:
-1. [Positive performance gap set](http://purl.obolibrary.org/obo/PSDO_0000117) - prefixIRI PSDO:0000117
-2. [Social comparator set](http://purl.obolibrary.org/obo/PSDO_0000045) - prefixIRI PSDO:0000045
-3. [Positive performance trend set](http://purl.obolibrary.org/obo/PSDO_0000120) - prefixIRI PSDO:0000120
-4. [Achievement set](http://purl.obolibrary.org/obo/PSDO_0000121) - prefixIRI PSDO:0000121
+1. [Social comparator element <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000045)
```
Comment (mackgalante) [inline vignettes/social_gain_vignette/readme.md:85]: Almost any time we list out bioportal conditions in the vignettes, I greatly prefer that the elements are ordered based on their BioPortal ID #, from smallest to largest. This is really just for consistency sake across the vignettes.

File: vignettes/social_gain_vignette/readme.md [hunk 2]
```diff
@@ -75,32 +75,31 @@ This pathway describes the influence of feedback interventions informing the rec
...
 Preconditions for social gain are factors necessary for the success of the precision feedback intervention using this pathway. The social gain pathway has the following preconditions:
-
 Information content preconditions:
-1. [Positive performance gap content](http://purl.obolibrary.org/obo/PSDO_0000104) - prefixIRI PSDO:0000104
-2. [Social comparator content](http://purl.obolibrary.org/obo/psdo_0000095) - prefixIRI PSDO:0000095
-3. [Positive trend content](http://purl.obolibrary.org/obo/PSDO_0000099) - prefixIRI PSDO:0000099
-4. [Achievement content](http://purl.obolibrary.org/obo/PSDO_0000112) - prefixIRI PSDO:0000112
+1. [Social comparator content <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000095)
+2. [Positive performance gap content <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000104)
+3. [Positive performance trend content <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000099)
```
Comment (mackgalante) [inline vignettes/social_gain_vignette/readme.md:81]: Need to make sure that as a team we are referencing things with the names they are given in bioportal and cedar, and not abbreviating things :)

File: vignettes/social_gain_vignette/readme.md [hunk 3]
```diff
@@ -109,35 +108,35 @@ A successful precision feedback intervention leveraging the social gain pathway
 ## Message templates
 Message templates represent possible motivational messages that a precision feedback system can send. There are three message templates that the precision feedback system can access for this vignette: 
 
-**[Top 10% Peer Benchmark](https://i.imgur.com/I1EeK7L.jpeg)** <sub>Todo - JSON template in Github, link here</sub> message:
+The `Achieved top 10% peer benchmark` [<sub>(GH)</sub>](https://github.com/Display-Lab/knowledge-base/blob/main/message_templates/achieved_top_10_peer_benchmark.json) message reads:
```
Comment (mackgalante) [inline vignettes/social_gain_vignette/readme.md:111]: Making sure message templates have consistent names across all platforms, including the vignettes

File: vignettes/worsening_vignette/readme.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Gaile, a resident anesthesiologist at Midwest Medicine, has the following perfor
...
+Preferences for precision feedback are elicited through a preference survey that providers can take. The preference survey produces a preference model for each provider that, with the provider's permission, is shared with MPOG to maintain. MPOG analyses preferences data that is shared to identify population-level preference segments. This generates preference profiles that can serve as a default preference model for an organization, or which can be selected by providers who do not take the preference survey, but who identify preferences that are close enough to their own in the settings menu for the precision feedback system.
```
Comment (mackgalante) [inline vignettes/worsening_vignette/readme.md:37]: Changed the wording here on only this vignette before I decided to elevate this to an issue for team review, per feedback from ZLL on 5/30

See issue #104

File: vignettes/social_gain_vignette/readme.md (hunk 2 above)
Comment (mackgalante) [reply vignettes/social_gain_vignette/readme.md:81]: Oops, github didn't highlight the difference like I wanted it to:
- changed [Positive trend content] to [Positive performance trend content] to match BioPortal entry

File: vignettes/social_gain_vignette/readme.md (hunk 1 above)
Comment (mackgalante) [reply vignettes/social_gain_vignette/readme.md:85]: Also changed the element name from `Social comparator set` to `Social comparator element` - reflects how they are named in BioPortal

File: vignettes/worsening_vignette/readme.md (hunk 4 above)
Comment (mackgalante) [reply vignettes/worsening_vignette/readme.md:37]: Wording updated to social loss with Zach 6/6, propagating out in new issue #104

**Review by zachll — APPROVED** [review]

**Pull request merged**